iocore/eventsystem/SocketManager.cc

```diff
@@ -108,3 +111,36 @@ SocketManager::close(int s)
   } while (res == -EINTR);
   return res;
 }
+
+bool
+SocketManager::fastopen_supported()
```

Comment:
Right. Is that likely to change for a given build? Isn't it a property of the host OS that's fixed?